CONTRIBUTORS.md

```diff
@@ -0,0 +1,42 @@
+OceanWATERS Team
+----------------
+
+K. Michael Dalal (project lead)
+Chetan Kulkarni
+Thomas Stucky
+Terence Welsh
+
+Alumni and Interns
+------------------
+
+Milan Addepalli
+Mark Allan
+John Bresina
+Liam Carr
+Damiana Catanoso
+Anjan Chakrabarty
+Laurence Edwards
+Jeremy Frank
+Chuck Fry
+Jason Fugate
+Leslie Keely
+Anthony Koutroulis
+Albert Kutsyy
+Lanssie Ma
+Ussama Naal
+Keegan Nave
+Molly O'Connor
+Arno Rogg
+Emily Schiede
+Mark Shirley
+Antoine Tardy
+Pranay Thangeda
+Orkan Umurhan
+Hans Utz
+Uland Wong
+Devin Wright
+Yinan Xu
+Honghau Zhen
+
+In addition, a number of external users have made valuable code
```

Comment:
It's common to include these names as well. You could have a separate category if you like.

Reply:
The truth is, only a handful of external users have made contributions, where were all small fixes.  I've just removed this sentence, because it would be work to create and maintain a list here.  If we ever get a substantial external contribution I think we should mention it here.